internal/daemon/api/grpc/api.pb.go

```diff
@@ -0,0 +1,626 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Me personally prefer to commit these autogenerated files, because it allows others to compile the code without the need of installing all the necessary tools (if the repo would be a library, we would not even discuss this topic, because not committing these files would prevent `go get` from working). It also makes the build pipeline on travis (or github actions) simpler, because otherwise you have to make all these tools available in the build pipeline as well in order to be able to build and execute the tests.

I think it does not do any harm to commit them.